buildSrc/src/main/java/org/opensearch/gradle/BwcVersions.java

```diff
@@ -268,6 +268,11 @@ private String getBranchFor(Version version) {
         // The current version is being worked, is always unreleased
         unreleased.add(currentVersion);
 
+        // No unreleased versions for 1.0.0
```

Comment:
Trying to wrap my head around this.
When we have 1.1.0 released how would this be?

Reply:
See my comment above